libs/cardano-data/cardano-data.cabal

```
@@ -52,13 +57,18 @@ library testlib
         base,
         containers,
         hspec,
-        QuickCheck
+        QuickCheck,
+        cardano-ledger-binary:testlib,
+        cardano-data
 
 test-suite cardano-data-tests
     type:             exitcode-stdio-1.0
     main-is:          Main.hs
     hs-source-dirs:   test
-    other-modules:    Test.Cardano.Data.MapExtrasSpec
+    other-modules:
+        Test.Cardano.Data.MapExtrasSpec
+        Test.Cardano.Data.OSet.Strict.Spec
```

Convention is adding a suffix, rather a separate module. (See above example `MapExtraSpec`). We should try to be consistent
```suggestion
        Test.Cardano.Data.OSet.StrictSpec
```